packages/server/src/handlers/BaseHandler.ts

```diff
@@ -1,12 +1,13 @@
 import EventEmitter from 'node:events'
 
 import type {ServerOptions} from '../types'
-import type {DataStore} from '../models'
+import type {DataStore, Upload} from '../models'
 import type http from 'node:http'
 
 const reExtractFileID = /([^/]+)\/?$/
 const reForwardedHost = /host="?([^";]+)/
 const reForwardedProto = /proto=(https?)/
+const own = Object.prototype.hasOwnProperty
```

Comment:
This looks unused. (aside: there should probably be a lint rule for that at some point)
```suggestion
```

Reply:
Updated the rule to `error`